commit 450f833febb0aecbf439e05f13e66e516e62af7b
Author: Mantas Mikulėnas <grawity@gmail.com>
Date:   Mon Aug 8 11:07:38 2016 +0300

    yummy:patchX:Revert--logind--really-handle--KeyIgnoreInhibited-options-in-logind-conf-.patch

Revert "logind: really handle *KeyIgnoreInhibited options in logind.conf"

    
    This reverts commit 8121f4d209eca85dcb11830800483cdfafbef9b7.
    
    The special 'key handling' inhibitors should always work regardless of
    any *IgnoreInhibited settings – otherwise they're nearly useless.
    
    Reverts: #3470
    Fixes: #3897
    (cherry picked from commit 06a70b918d4d753769a727239f75af8896006467)

diff --git a/src/login/logind-action.c b/src/login/logind-action.c
index 8ef48db..a950409 100644
--- a/src/login/logind-action.c
+++ b/src/login/logind-action.c
@@ -85,7 +85,7 @@ int manager_handle_action(
         }
 
         /* If the key handling is inhibited, don't do anything */
-        if (!ignore_inhibited && inhibit_key > 0) {
+        if (inhibit_key > 0) {
                 if (manager_is_inhibited(m, inhibit_key, INHIBIT_BLOCK, NULL, true, false, 0, NULL)) {
                         log_debug("Refusing operation, %s is inhibited.", inhibit_what_to_string(inhibit_key));
                         return 0;
